docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,418 @@
+
+<figure>
+  <img width="512" height="768" alt="An example of a Data Grid's list view on a phone" src="/static/blog/mui-x-v8/list-view.png" loading="lazy">
+  <figcaption>An example of a Data Grid's list view on a phone</figcaption>
+</figure>
+
+Learn more about [List view](/x/react-data-grid/list-view/).
+
+### Row spanning
+
+Row spanning has also been promoted to stable, providing a reliable and flexible solution for merging cells across multiple rows.
+This enhancement makes it easier to create complex, multi-row layouts within your Data Grid, allowing you to present your data more effectively.
+
+Learn how to set up and use [Row spanning](/x/react-data-grid/row-spanning/).
+
+### Server-side Aggregation and Lazy loading [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+The Data Source now supports advanced features such as server-side aggregation, on-demand data fetching for infinite scrolling, and lazy loading.
+
+Explore all your options when handling [Server-side data](/x/react-data-grid/server-side-data/).
+
+### Data source on the Community Plan
+
+Great news—the Data Source move from the Pro Plan to the Community Plan (MIT licensed)!
+
+We're unifying our approach to handling server-side data across all plans, creating a single, cohesive framework.
+
+Explore your options in our [Data source docs](/x/react-data-grid/server-side-data/).
+
+### New Toolbar
+
+The [redesigned toolbar](/x/react-data-grid/components/toolbar/) not only looks fantastic but also empowers developers one step further through an open composition of subcomponents, allowing you to tailor the toolbar's layout precisely to your application's needs.
+
+<figure>
+  <img width="1170" height="104" alt="The new toolbar's design" src="/static/blog/mui-x-v8/new-toolbar.png" loading="lazy" style="border: 0;">
+  <figcaption>The new toolbar's design</figcaption>
+</figure>
+
+This new developer experience, based on composition, may feel different from our previous approach—but it sets a new standard.
+We'll soon extend this enhanced experience to other key subcomponents, such as the filter panel and columns management panel, ensuring a cohesive and powerful toolkit for advanced customization.
+
+Dive into our new docs section on [Components usage](/x/react-data-grid/components/usage/).
+
+### Design agnostic core
+
+We understand customization is important to you.
+So with the ultimate goal of being fully customizable, we're taking the first step to allow using the Data Grid component without a single Material UI dependency (for example
+without Emotion), opening the possibility to use the Data Grid with any design system—and this is just the beginning.
+
+Next up, we'll work to provide the best experience for you to build a full-feature Data Grid that fits all your design requirements.
```

Comment:
I think to link to the docs or to remove. If we don't have docs yet, removing it seems much better, so we can properly introduce it in a v8.x minor version.